### PR TITLE
[Settings] Create New Settings View

### PR DESCRIPTION
**Pull request by shubhamkmr04**
This PR is to create the new settings view. I did some work on the visuals and this is how it looks right now.
Screenshot from 2021-09-03 15-20-38

### CODE REVIEW

**Comment by shubhamkmr04** [comment]
@kaloudis  Added the icons.

Screenshot from 2021-09-03 20-17-01

**Comment by shubhamkmr04** [comment]
Screenshot from 2021-09-05 16-09-46
@kaloudis does it look good?

File: views/Settings.tsx [hunk 1]
```diff
@@ -221,26 +231,216 @@ export default class Settings extends React.Component<
...
+                        alignSelf: 'center',
+                        bottom: 25,
+                        position: 'absolute'
+                    }}
+                >
+                    Zeus version 0.5.2
+                </Text>
 
-                <DropdownSetting
+                {/* <DropdownSetting
```
Comment (kevkevinpal) [inline views/Settings.tsx:434]: Did we want to remove the commented items?

Comment (shubhamkmr04) [reply views/Settings.tsx:434]: No, I have just temporarily commented this. I was just working on visuals of this view, will be adding the functionality in the upcoming commits.

**Comment by kaloudis** [comment]
Created a new feature branch named `settings` for us to work off of

**Review by kaloudis — APPROVED** [review]

**Pull request merged**